### PR TITLE
Update to RBAC section of docs

### PR DESCRIPTION
**Pull request by aelkugia**
Several customers have asked about updating role mapping on an existing trusted cluster. Just added a note on this in the trustedclusters.md

### CODE REVIEW

File: docs/4.0/quickstart.md [hunk 1]
```diff
@@ -238,14 +238,14 @@ on all servers located in Virginia:
...
 
-## Sharing SSH Sessions 
+## Sharing SSH Sessions
 
-Suppose you are trying to troubleshoot a problem on a node. Sometimes it makes sense to ask 
-another team member for help. Traditionally this could be done by letting them know which 
-node you're on, having them SSH in, start a terminal multiplexer like `screen` and join a 
+Suppose you are trying to troubleshoot a problem on a node. Sometimes it makes sense to ask
+another team member for help. Traditionally this could be done by letting them know which
+node you're on, having them SSH in, start a terminal multiplexer like `screen` and join a
```
Comment (benarent) [inline docs/4.0/quickstart.md]: @aelkugia it looks like you have your IDE setup to remove tailing spaces. While we don't have a strong guidelines in the docs, it makes PR reviews harder to review.

File: docs/4.0/trustedclusters.md [hunk 2]
```diff
@@ -144,6 +144,7 @@ Consider the following:
 * Every user in Teleport Enterprise is assigned a role.
 * When creating a _trusted cluster_ resource, the administrator of "east" must
   define how roles from "main" map to roles on "east".
+* To update role map for an existing _trusted cluster_ delete and re-create the _trusted cluster_ with the updated role map
```
Comment (benarent) [inline docs/4.0/trustedclusters.md]: I would consider also adding a section after the Example, that is dedicated section on Updating trust. Between the last RBAC Trusted Cluster and `Using Trusted Clusters` 
image 

Possibly with `tctl rm main-cluster.yaml` -> `tctl create main-user-updated-role.yaml` as an example.

File: docs/4.0/quickstart.md (hunk 1 above)
Comment (aelkugia) [reply docs/4.0/quickstart.md]: Weird the IDE setting had that unchecked. I'll keep an eye out for this in future
whitespace
 PRs.

File: docs/4.0/trustedclusters.md (hunk 2 above)
Comment (aelkugia) [reply docs/4.0/trustedclusters.md]: Got it, just made this update

**Review by benarent — APPROVED** [review]
Looks good, FYI looks like some merge conflicts.

**Pull request merged**